spreadsheet_dashboard_oca/models/spreadsheet_dashboard.py

```diff
@@ -42,3 +43,24 @@ def _compute_can_edit(self):
             )
             if imd and imd.module != "__export__":
                 record.can_edit = imd.noupdate
+
+    @api.model
+    def _search_can_edit(self, operator, value):
+        if operator != "=":
+            raise NotImplementedError(_("Search operation not supported"))
+        if not isinstance(value, bool):
+            raise ValidationError(_("The value has to be a boolean"))
+        no_edit_ids = (
+            self.env["ir.model.data"]
+            .search(
+                [
+                    ("model", "=", self._name),
+                    ("module", "!=", "__export__"),
+                    ("noupdate", "=", 0),
+                ]
+            )
+            .mapped("res_id")
+        )
```

Comment:
I created a brand new dashboard but I can't make it to show up in the wizard. Am I doing something wrong

Reply:
You have to add a spreadsheet to the panel to be able to show it

Reply:
I mean, I can't select any dashboard in the wizard

Reply:
![example](https://github.com/user-attachments/assets/c3c51ead-6cb6-4a48-b7c1-b5d2c673c110)

I am able to select it.

Reply:
I was trying to dashboard new sheet. I was expecting to let me choose a dashboard to wich apend a sheet. Did I understand it wrong?

Reply:
Yes :sweat_smile: that option is to add a sheet to a created spreadsheet for the dashboard

Example:
![new sheet example](https://github.com/user-attachments/assets/d0d040cb-5f2d-4452-8bca-444abf4dc58b)

This option can be used set data that has to be hidden but it has to be used to make some operations

Reply:
I think that option that you say can be set, I will see how to add it 

Thanks :smile:

Reply:
Ah, ok. See? I was getting it wrong :sweat_smile:

Reply:
Added new option to create new dashboards :smile: